### PR TITLE
feat: add a11y test for components demo

### PR DESCRIPTION
**Pull request by aojunhao123**
### 🤔 This is a ...

- [x] 🆕 New feature
- [x] 📽️ Demo improvement
- [x] ✅ Test Case

### 🔗 Related Issues

> - related to #51348 

### 💡 Background and Solution

> - 为了确保组件库的可访问性符合标准，添加了自动化的可访问性测试工具。
> - 为所有组件的demo添加可访问性测试

### 📝 Change Log

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     •Added automated accessibility testing tools to ensure the component library meets accessibility standards. <br/>•Added accessibility tests for all component demos to ensure compliance with accessibility guidelines.      |
| 🇨🇳 Chinese |  •为了确保组件库的可访问性符合标准，添加了自动化的可访问性测试工具。<br/>•为所有组件的示例（demo）添加了可访问性测试，确保符合无障碍标准。         |

### CODE REVIEW

**Comment by aojunhao123** [comment]
@afc163

File: tests/shared/accessibilityTest.tsx [hunk 1]
```diff
@@ -1,15 +1,148 @@
...
+      ...acc,
+      [rule]: { enabled: false },
+    }),
+    {},
+  );
+};
+
 // eslint-disable-next-line jest/no-export
-export default function accessibilityTest(Component: React.ComponentType) {
+export default function accessibilityTest(
```
Comment (zombieJ) [inline tests/shared/accessibilityTest.tsx:63]: 这么看起来这个不需要 default export？

Comment (aojunhao123) [reply tests/shared/accessibilityTest.tsx:63]: 需要的，个别组件的单测里面用到了，我理解后续是不是要把a11y测试都加到组件的单测里面
QQ_1730274407758

Comment (zombieJ) [reply tests/shared/accessibilityTest.tsx:63]: 那是不是 accessibilityDemoTest 作为 default export 会更好些？

**Review by zombieJ — APPROVED** [review]

**Comment by aojunhao123** [comment]
related to https://github.com/ant-design/ant-design/pull/51372

**Pull request merged**